### PR TITLE
Display roles for organization listed when using non-English web browser

### PR DESCRIPTION
**Pull request by nixocio**
Display roles for organization listed when using non-English web browser

### CODE REVIEW

File: awx/ui/src/components/ResourceAccessList/ResourceAccessList.js [hunk 1]
```diff
@@ -56,31 +56,29 @@ function ResourceAccessList({ apiModel, resource }) {
...
-        });
+        const [
+          {
+            data: { results: systemAdmin },
+          },
+          {
+            data: { results: systemAuditor },
+          },
+        ] = await Promise.all([
+          RolesAPI.read({ singleton_name: 'system_administrator' }),
```
Comment (marshmalien) [inline awx/ui/src/components/ResourceAccessList/ResourceAccessList.js:67]: Nice! I didn't realize we could query this directly - this lifts some of the complexity from the UI code.

**Review by marshmalien — APPROVED** [review]

File: awx/ui/src/components/ResourceAccessList/ResourceAccessList.js (hunk 1 above)
Comment (nixocio) [reply awx/ui/src/components/ResourceAccessList/ResourceAccessList.js:67]: That was @AlanCoding magic. 🧙

**Comment by nixocio** [comment]
I triggered the e2e tests.

**Review by InnocentK — APPROVED** [review]
Fix looks good. Running some E2E tests and can merge after verifying those results

**Pull request merged**